pkg/bpf/bpf_linux.go

```diff
@@ -52,11 +56,26 @@ func objCheck(m *ebpf.Map, path string, mapType ebpf.MapType, keySize, valueSize
 		mismatch = true
 	}
 
+	if mi, err := m.Info(); err != nil {
+		scopedLog.Error("Could not get mapinfo for map %+v", *m)
```

Comment:
The linter points it out already, but just to be sure, this should be an `Errorf` not an `Error` given the format option